commcare_connect/opportunity/tests/test_visit_import.py

```diff
@@ -25,6 +32,30 @@ def test_bulk_update_visit_status():
     assert after_status == {VisitValidationStatus.approved.value}
 
 
+@pytest.mark.django_db
```

Comment:
nit: fun fact that simon taught me is that you don't need these annotations if your test depends on fixtures that depend on the `db` fixture, as these two fixtures do